operator/src/test/java/org/keycloak/operator/testsuite/integration/ClusteringTest.java

```diff
@@ -45,22 +51,74 @@
 @QuarkusTest
 public class ClusteringTest extends BaseOperatorTest {
 
+    @Test
+    public void testMultipleDeployments() throws InterruptedException {
+        // given
+        var kc = K8sUtils.getDefaultKeycloakDeployment();
+
+        // another instance running off the same database
+        // - should eventually give this a separate schema
+        var kc1 = K8sUtils.getDefaultKeycloakDeployment();
+        kc1.getMetadata().setName("another-example");
+        kc1.getSpec().getHostnameSpec().setHostname("another-example.com");
+        // this is using the wrong tls-secret, but simply removing http spec renders the pod unstartable
```

Comment:
jFYI – you can disable TLS as:
https://github.com/keycloak/keycloak/blob/58713a23b2b39f55e97bbadee95cceb7c3d3606c/operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakIngressTest.java#L67-L71
But no need for it here, it's fine as is.